cluster-controller/src/main/java/io/strimzi/controller/cluster/model/AbstractModel.java

```diff
@@ -57,6 +59,9 @@
     private static final Long VOLUME_MOUNT_HACK_GROUPID = 1001L;
 
     public static final String METRICS_CONFIG_FILE = "config.yml";
+    public static final String ENV_VAR_DYNAMIC_HEAP_FRACTION = "DYNAMIC_HEAP_FRACTION";
+    public static final String ENV_VAR_KAFKA_HEAP_OPTS = "KAFKA_HEAP_OPTS";
+    public static final String ENV_VAR_DYNAMIC_HEAP_MAX = "DYNAMIC_HEAP_MAX";
```

Comment:
who is going to use these `DYNAMIC_HEAP_FRACTION` and `DYNAMIC_HEAP_MAX` env vars ? a script ?

Reply:
Yes, [`dynamic_heap.sh`](https://github.com/strimzi/strimzi/pull/390/files#diff-7b3b36b30cce4f553fa0544450381aba) consumes these, figures out the approx maximum heap for a jvm which would use all the memory upto the limit, and then does:

    max(heap_max, fraction*max_heap_for_limit)

This allows us to cap Kafka heap at 5g, for example, but otherwise be sensitive to the limit.